src/smart-components/role/add-role-new/add-role-wizard.scss

```diff
@@ -42,3 +42,15 @@
         }
     }
 }
+
+.ins-c-rbac-permissions-table {
+    .ins-c-primary-toolbar {
+        .pf-c-toolbar__content {
+            .pf-c-toolbar__content-section {
+                .pf-m-filter-group {
+                    max-width: 35em;
```

Comment:
How responsive is this? I think that using a % value might be a bit safer.

Reply:
I have experiment a bit with media queries and it seems to be working well now. Even though it is a bit complex. 
Unfortunately, github does not support `.webm` files https://webm.red/view/YtI4.webm
(I can't make gifs)

Reply:
Yeah that looks good. Can you fix up travis?